doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1154 @@
+       "                <td style=\"text-align: left\"><b>Ray version:</b></td>\n",
+       "                <td style=\"text-align: left\"><b> 2.0.1</b></td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "    <td style=\"text-align: left\"><b>Dashboard:</b></td>\n",
+       "    <td style=\"text-align: left\"><b><a href=\"http://127.0.0.1:8265\" target=\"_blank\">http://127.0.0.1:8265</a></b></td>\n",
+       "</tr>\n",
+       "\n",
+       "        </table>\n",
+       "    </div>\n",
+       "</div>\n"
+      ],
+      "text/plain": [
+       "RayContext(dashboard_url='127.0.0.1:8265', python_version='3.8.13', ray_version='2.0.1', ray_commit='03b6bc7b5a305877501110ec04710a9c57011479', address_info={'node_ip_address': '127.0.0.1', 'raylet_ip_address': '127.0.0.1', 'redis_address': None, 'object_store_address': '/tmp/ray/session_2022-11-08_01-31-56_114290_77509/sockets/plasma_store', 'raylet_socket_name': '/tmp/ray/session_2022-11-08_01-31-56_114290_77509/sockets/raylet', 'webui_url': '127.0.0.1:8265', 'session_dir': '/tmp/ray/session_2022-11-08_01-31-56_114290_77509', 'metrics_export_port': 65423, 'gcs_address': '127.0.0.1:61483', 'address': '127.0.0.1:61483', 'dashboard_agent_listen_port': 52365, 'node_id': 'd5d881de21f7529000c167d2bf85209429ee22e41e935f5dada35598'})"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ray\n",
+    "\n",
+    "if ray.is_initialized():\n",
+    "    ray.shutdown()\n",
+    "ray.init()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ac0af2c6-7c90-42bc-8687-3c9b2dc9a993",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For benchmarking purposes, we can print the times of various operations.\n",
+    "# In order to reduce clutter in the output, this is set to False by default.\n",
+    "PRINT_TIMES = False\n",
+    "\n",
+    "def print_time(msg: str):\n",
+    "    if PRINT_TIMES:\n",
+    "        print(msg)\n",
+    "        \n",
+    "# To speed things up, we’ll only use a small subset of the full dataset consisting of two last months of 2019.\n",
+    "# You can choose to use the full dataset for 2018-2019 by setting the SMOKE_TEST variable to False.\n",
+    "SMOKE_TEST = True\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26d69835-0104-4a89-8418-b1986e45a15c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Load and Prepare Parquet data <a class=\"anchor\" id=\"load_data\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d558f234-b7af-489d-8aaa-42a7c49f2194",
+   "metadata": {},
+   "source": [
+    "Apache Arrow uses a [C++ implementation of Apache Parquet](http://github.com/apache/parquet-cpp), which includes a native, multithreaded C++ adapter. This makes **PyArrow dataset and table faster for large parquet file reading than pandas read_parquet, even with engine=pyarrow**. \n",
```

Comment:
we can make this a tip:
````
```{tip}
tip text
```
````

in general this is just an implementation detail so let's not spend too much text on it. the most important thing is what we are doing here (loading the data) and not the tool we are using